src/interface/common.thrift

```diff
@@ -112,5 +112,50 @@ struct Pair {
     2: string value,
 }
 
+/**
+** GOD is A global senior administrator.like root of Linux systems.
+** ADMIN is an administrator for a given Graph Space.
+** DBA is an schema administrator for a given Graph Space.
+** USER is a normal user for a given Graph Space. A User can access (read and write) the data in the Graph Space.
+** GUEST is a read-only role for a given Graph Space. A Guest cannot modify the data in the Graph Space.
+** Refer to header file src/graph/PermissionManager.h for details.
+**/
+
+enum RoleType {
+    GOD    = 0x01,
+    ADMIN  = 0x02,
+    DBA    = 0x03,
+    USER   = 0x04,
+    GUEST  = 0x05,
+} (cpp.enum_strict)
+
+enum UserLoginType {
+   PASSWORD = 0x01,
+   LDAP     = 0x02,
+}
+
+struct RoleItem {
+    1: string        user,
+    2: string        space,
+    3: RoleType      role_type,
+}
+
+struct UserItem {
+    1: required string        account;
```

Comment:
Do we have id for user?

Reply:
> Do we have id for user?

I've removed user id . because we can using user name to keep the uniqueness for users.